bin/bundle

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env ruby
```

Comment:
Where'd all of these `bin/bundle` files come from?

Not sure we need it in version control.